### DISCUSSION TITLE
Mac or CLI version ?

### DISCUSSION THREAD

**Discussion by charafsalmi** [audiamus/AaxAudioConverter#188] [General]
Hi,

Is there a mac version planned? I tried to make it work using Wine, but without any success.

Or maybe a simple Unix (meaning any mac/linux) CLI version?

Best regards.

**Comment by audiamus** [comment]
I'm afraid not. Linux or Mac would require a completely new platform independent user interface. AAX Audio Converter and Book Lib Connect are written with Windows Forms. It's the easiest package to work with and actually quite powerful if you know how your way around. But it's Windows only. For the the other platforms it would mean Avalonia instead, and Avalaonia is, like all XAML-based toolkits, much more complex, if you want to do it bit more than just basic stuff. I had to work with Avalonia for a Linux project but never became friends with it, despite all its appealing theory like MVVM and ReactiveUI.  

_I will convert this topic to the discussion section._

**Reply by audiamus** [reply]
_I forgot:_ 

Command Line Interface. I don't see the need for this, as both AAX Audio Converter and Book Lib Connect are designed for batch mode. Manually gathering command line arguments or setting up an ini file of some kind is much more work for the user than just clicking a few books from a list for processing in a GUI.
